types/intercom-client/Tag.d.ts

```diff
@@ -0,0 +1,30 @@
+import { UserIdentifier } from "./User";
+
+export interface TagIdentifier {
+  id: string
+}
+
+export interface Tag extends Partial<TagIdentifier> {
+  readonly type: "tag",
+  readonly id: string,
+  name: string,
+}
+
+export interface List {
+  "type": "tag.list",
+  "total_count": number,
+  "tags": (Tag)[],
+  "pages": { "next"?: string, "page": number, "per_page": number, "total_pages": number }
+}
+
+interface TagUsers {
+  name: string,
+  users: UserIdentifier[]
+}
+
+interface TagCompanies {
+  name: string,
+  users: { id: string }[]
```

Comment:
Hmm, is this supposed to be `companies` instead of `users`, or am I just using the client wrong? 😅